server/serverBundleEntry.js

```diff
@@ -0,0 +1,3 @@
+import Community from '../client/src/components/community.jsx';
+
+module.exports = { Community };
```

Comment:
I don't think destructuring this is required as you are doing a default export

Reply:
This isn't destructuring, it's wrapping it in an object - but that's actually unnecessary and confusing, so thanks for drawing my attention to it. Fixed.